vast.conf

```diff
@@ -1,116 +1,380 @@
+; This is an example configuration file for VAST, striving to show all available
+; options. Lines starting with a semicolon are commented out. Options in angle
+; brackets have their default value determined at runtime.
+
+; Options that apply to VAST.
 system {
-;; The host and port to listen at and connect to.
-; endpoint = "localhost:42000"
+  ; The host and port to listen at and connect to.
+  ;endpoint = "localhost:42000"
+
+  ; The file system path used for persistent state.
+  ;db-directory = "vast.db/"
+
+  ; The file system path used for log files.
+  ;log-file = "vast.db/server.log"
```

Comment:
Is this taking the variable `db-directory` or the hard-coded string `vast.db`?

Reply:
It's `<db-directory>/server.log` by default. I should write it like that.